ipams/builtin/builtin_unix.go

```diff
@@ -41,3 +46,8 @@ func Init(ic ipamapi.Callback, l, g interface{}) error {
 
 	return ic.RegisterIpamDriverWithCapabilities(ipamapi.DefaultIPAM, a, cps)
 }
+
+// SetDefaultIPAddressPool stores default address pool to local variable.
+func SetDefaultIPAddressPool(addressPool []*ipamutils.PredefinedPools) {
```

Comment:
we should have a corresponding Get and Use it above. So that ppl adding code are not using the variable . 
Also For a global variable its a good practice to use sync.Once to set the variable to avoid race condition. I don't think there is a possibility for it. but its safer to do it that way. 
May be we can ignore the sync.Once for this , because networkinit is under that umbrella.

Reply:
added get function